data/chapters/mexico-​chiapas-tuxtla.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "urlname": "rladies-tuxtla-gutierrez",
+    "status": "prospective",
+    "country": "mexico",
+    "city": "tuxtla-gutierrez",
+    "social_media": {
+      "meetup": "rladies-tuxtla-gutierre",
+      "email": "tuxtla@rladies.org"
+    },
+    "organizers": {
+      "current": ["comma separated", "Amin Siman Israel"]
```

Comment:
```suggestion
      "current": ["Amin Siman Israel"]
```

Reply:
@MounaBelaid awesome if you can remove the "comma separated" part i the future :)

Reply:
Yes, sure! Thanks.